third_party/libusb/naclport/src/libusb_over_chrome_usb.cc

```diff
@@ -92,16 +92,15 @@ constexpr int kLibusbRequestTypeMask =
     LIBUSB_REQUEST_TYPE_STANDARD | LIBUSB_REQUEST_TYPE_CLASS |
     LIBUSB_REQUEST_TYPE_VENDOR | LIBUSB_REQUEST_TYPE_RESERVED;
 
-std::unique_ptr<uint8_t> CopyRawData(const uint8_t* data, size_t byte_count) {
-  std::unique_ptr<uint8_t> result(new uint8_t[byte_count]);
+std::unique_ptr<uint8_t[]> CopyRawData(const uint8_t* data, size_t byte_count) {
```

Comment:
(Fixed it while we're here. Technically, the old code was correct as well, since we never destroy those `std::unique_ptr`, but always do `.release() + delete[]`. But it was probably confusing and not future-proof to omit these brackets here.)